ls1intum/Artemis
# Saving result in database after using trigger all functionality

### Checklist

### Motivation and Context


Resolves https://github.com/ls1intum/Artemis/issues/3689

### Description

Fixing the linked issue https://github.com/ls1intum/Artemis/issues/3689 by saving the result instead of the submission.

### Steps for Testing

You will need a manually assessed programming exercise with at least one manually assessed participation.

1. Create a programming exercise with manual assessment
2. Participate as a student
3. Assess the programming exercise manually
4. Change the grading of the respective exercise (e.g. change the weight, add bonus points to a test, etc.)
5. Check whether the changes are displayed correctly after reloading the scores page (this may take a few minutes depending on the capacity of the CIS)

Further interesting scenarios:
- Repeat the steps for an exam programming exercise.
- Change the tests (e.g. a student could now get a build failed)

### Screenshots


The score of the semi-automatic assessment should be adapted properly after triggering changed tests.
![image](https://user-images.githubusercontent.com/63976129/124810531-bebfe180-df61-11eb-86dd-e58457939f79.png)
